node/node.go

```diff
@@ -437,7 +437,7 @@ func (node *Node) validateShardBoundMessage(
 
 	if !node.Consensus.IsValidatorInCommittee(senderKey) {
 		atomic.AddUint32(&node.NumInvalidMessages, 1)
-		return nil, nil, errors.WithStack(shard.ErrValidNotInCommittee)
+		return &m, nil, errors.WithStack(shard.ErrValidNotInCommittee)
```

Comment:
Why are we returning message?

Reply:
The message can be used later to debug and check why this node is not in the current committee.  It can be helpful when we determine if we need to blacklist the message sender.